packages/orchestration/test/chain-info.test.js

```diff
@@ -0,0 +1,28 @@
+import test from 'ava';
+import { makeAgoricNamesAccess } from '@agoric/vats';
+import { registerChainNamespace } from '../src/chain-info.js';
+
+test('chain-info', async t => {
+  const { agoricNames, agoricNamesAdmin } = await makeAgoricNamesAccess(t.log);
+  await registerChainNamespace(agoricNamesAdmin, t.log);
```

Comment:
again, I don't see how this registration can be build time, using mainnet info. I think we need to support test networks. Seems like all but the canonical transfer channel info needs to come from `agd query ...` or the like.

Reply:
chain-registry provides info for mainnet, testnet and devnet. So we can support test networks using that data source.

But I don't see that requirement as in scope for this PR.